spec/metrics/scoping_spec.rb

```diff
@@ -58,7 +58,7 @@
 
     it "works with several scopes" do
       expect(doomy.where(dummy_id: 100).by_user([1, 2, 3]).daily.to_sql)
-        .to eq "select * from \"dummy\" group by time(1d) where (dummy_id=100) and (user_id=1 or user_id=2 or user_id=3) limit 100"
+        .to eq "select * from \"dummy\" where (dummy_id=100) and (user_id=1 or user_id=2 or user_id=3) group by time(1d) limit 100"
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver.